src/libraries/System.Net.Security/src/System/Net/Security/SslApplicationProtocol.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Are these still needed if they aren't in the `Common` folder?

Reply:
Many implementation files are referenced by the unit test project directly, which is not annotated. This is one of those files.

Reply:
Makes sense. Thanks.